x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/components/search_ui_graphic.scss

```
@@ -185,4 +185,27 @@
       }
     }
   }
+  &.activeThumb {
+    #results {
+      .outerBox {
+        fill: $euiColorEmptyShade;
+        stroke: $euiColorPrimary;
+        stroke-width: 1px;
```

[suepr optional]
```suggestion
        stroke-width: $euiBorderWidthThin;
```

Also, really cool that our svg was all set up for this! 🎉

---

Right? I think we got lucky 😄